Clinical-Genomics/scout
# General report shows only number of affected genes for dismissed SVs

Fix #2511 

**How to test**:
1. Dismiss a number of structural variants for a case (big variants encompassing many genes are especially good for testing)
2. Go to general report and check the different between this branch and master
3. Dismissed SNVs should look exactly the same as in master branch.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
